tests/test_exporter.py

```diff
@@ -61,10 +65,14 @@ def test_param_from_add_span_attributes_only_captured(exporter):
     )
     def decoratee(param):
         add_span_attributes({"added_param": 45})
+        add_span_attributes({"none_param": None})  # type: ignore
 
-    with asserting_span_exporter(exporter, "decoratee", "added_param"):
+    with asserting_span_exporter(exporter, "decoratee", "added_param", "none_param"):
         decoratee(1)
         exporter.force_flush()
 
     span = exporter.top_span.result(timeout=0.0)
-    assert len(span.attributes.keys()) == 1
+    assert span.attributes
+    assert span.attributes["added_param"] == 45
+    assert span.attributes["none_param"] == "None"
+    assert len(span.attributes.keys()) == 2
```

Comment:
```suggestion
    assert span.attributes
    assert span.attributes == {"added_param": 45, "none_param": "None"}
```

Reply:
I think it's better to keep the asserts separate so that it's immediately clear where the failure is if one occurs